docs/UserGuide.md

```diff
@@ -173,8 +173,26 @@ room <matric_number>
 A0123456X is allocated room <room_number>
 ```
 
+#### 5. Edit a room allocation for a student
```

Comment:
These changes appear to be more relevant for #46 . Perhaps they should not be included in this PR?

Reply:
Will resolve via rebase strategy.